tests/components/nest/conftest.py

```diff
@@ -76,3 +97,100 @@ def cleanup_media_storage(hass):
     with patch("homeassistant.components.nest.media_source.MEDIA_PATH", new=tmp_path):
         yield
         shutil.rmtree(hass.config.path(tmp_path), ignore_errors=True)
+
+
+@pytest.fixture
```

Comment:
Note to reviewer: When reviewing this file, you can compare to https://github.com/home-assistant/core/blob/dba65623564aee50b5fd8cb4b0e16ee0aa119551/tests/components/nest/common.py#L98 which is the original setup method used by most tests.